src/Resources/app/administration/src/module/ui-feed-export/component/ui-feed-export-form/index.js

```diff
@@ -23,6 +19,27 @@ Component.register('ui-feed-export-form', {
     ],
 
     methods: {
+        getExportTypeValues() {
+            let values = {};
+            const httpClient = Shopware.Service('syncService').httpClient;
+            let url = '_action/fact-finder/get-export-type-options';
+            const basicHeaders = {
+                Authorization: `Bearer ${Shopware.Context.api.authToken.access}`,
+                'Content-Type': 'application/json'
+            };
+
+            httpClient
+                .get(url, {
+                    headers: basicHeaders
+                })
+                .then((response) => {
+                    if (response.status === 200) {
+                        values = response.data;
```

Comment:
The response gets discarded because you synchronously return `values` with its initial value (`{}`).
When the request returns, the result is assigned to the _local_ `values` variable without leaving the current scope.
You will have to return the entire promise to make the result available to callers.